config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Tests run without needing to ease into it - 6.1. defaults work for us out of the box. (In other words this is properly all commented out and we can delete it in a future PR; mostly kept in version control here for reference.)